src/components/Profile.js

```diff
@@ -1,33 +1,46 @@
 import { Avatar, Box, Typography } from '@mui/material';
 import { useAuthContext } from '~/contexts/hooks';
 import useImage from '~/hooks/useImage';
+import theme from '~/theme';
 
 const DEFAULT_IMAGE_URL = '/images/logo.png';
-export default function Profile(props) {
+
+export default function Profile({ ...props }) {
   const { session } = useAuthContext();
   const imageUrl = useImage(session?.imgId);
+
   return (
     <Box sx={{ display: 'flex', gap: 1 }}>
       {/* Full Name */}
-      <Typography sx={{ justifyContent: 'center', alignContent: 'center' }}>
+      <Typography
+        sx={{ justifyContent: 'center', alignContent: 'center', fontSize: 12 }}
+      >
         {session
-          ? `${session.firstName} ${session.middleName} ${session.lastName}`
+          ? `${session.lastName} ${session.middleName} ${session.firstName}`
           : 'Guest'}
       </Typography>
       {/* Avatar */}
-      <Avatar
-        src={session ? imageUrl : DEFAULT_IMAGE_URL}
-        alt={
-          session
-            ? `${session.firstName} ${session.middleName} ${session.lastName}`
-            : 'Avatar'
-        }
+      <Box
         sx={{
-          objectFit: 'cover',
-          objectPosition: 'center',
+          backgroundImage: theme.palette.bright.blue.linear,
```

Comment:
no need to use useTheme hook to access theme when style for MUI component, try to shorten this syntax:
`bright.blue.linear`

Reply:
em đâu có sài thằng useTheme đâu

Reply:
để ý mới thấy :), import còn ác nữa. Đã có theme provider bọc lại rồi nên k chơi import vậy. 1 là xài hook của nó 2 là lấy từ prop, 3 là xài trực tiếp, đúng key nó tự parse ra value

Reply:
chọn 1 convention dùng chung luôn. Như a thì đang dùng kiểu 3 rồi